Recruit-Training/Advanced-Recruit-Training/src/BatteryStateDisplayService.cpp

```diff
@@ -25,9 +26,29 @@ void BatteryStateDisplayService::handleBatteryDataReceived(const BatteryData& ba
 {
     batteryStateOfChargeService_.addData(batteryData);
 
+    double milisec = batteryStateOfChargeService_.timeWhenChargedOrDepleted().msec();
+    QString addZero;
+    if(milisec < 100){
+        addZero = "0";
```

Comment:
Instead of padding it like this, try formatting your output instead:
https://stackoverflow.com/questions/2618414/convert-an-int-to-a-qstring-with-zero-padding-leading-zeroes